spec/utils/bsonUrlEncodingSpec.js

```diff
@@ -27,7 +27,7 @@ describe('bson url encoding', (it) => {
 
     t.is(
       str,
-      'eyJfaWQiOnsiJG9pZCI6IjU4MTY0ZDg2ZjY5YWI0NTk0MmM2ZmYzOCJ9LCJkYXRlIjp7IiRkYXRlIjoiMjAxNi0xMC0zMFQxOTozMjozNS4wMDBaIn0sIm51bWJlciI6MSwic3RyaW5nIjoiY29tcGxleCBTdHJpbmcgJiQjIyQtLz8ifQ'
+      'eyJfaWQiOnsiJG9pZCI6IjU4MTY0ZDg2ZjY5YWI0NTk0MmM2ZmYzOCJ9LCJkYXRlIjp7IiRkYXRlIjoiMjAxNi0xMC0zMFQxOTozMjozNVoifSwibnVtYmVyIjoxLCJzdHJpbmciOiJjb21wbGV4IFN0cmluZyAmJCMjJC0vPyJ9'
```

Comment:
Looks like the only real change here is that `bson` encodes the date without the millisecond resolution.

Reply:
`bson` does preserve millisecond resolution, so I think this is fine.
E.g. 
```
bson.EJSON.stringify(new Date('Sun Oct 30 2016 12:32:35.555 GMT-0700 (PDT)')) === '{"$date":"2016-10-30T19:32:35.555Z"}'
```